ReactWindows/ReactNative.Shared/Bridge/Queue/MessageQueueThread.cs

```diff
@@ -151,9 +153,9 @@ public DispatcherMessageQueueThread(string name, Action<Exception> handler)
             {
                 _actionSubject = new Subject<Action>();
                 _actionObserver = _actionSubject;
-#if NET46
+#if WINDOWS_UWP
```

Comment:
It's only the final diff that counts ;-).

Did you consider...
```
    _subscription = _actionSubject
#if WINDOWS_UWP
        .ObserveOnDispatcher()
#else
        .ObserveOn(Dispatcher.CurrentDispatcher)
#endif
       ...
```

---
In reply to: [82666295](https://github.com/ReactWindows/react-native-windows/pull/755#discussion_r82666295) [](ancestors = 82666295)